STUDIO/org.openl.rules.repository.git/pom.xml

```diff
@@ -33,5 +33,10 @@
             <artifactId>org.openl.rules.repository</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.openl.rules</groupId>
+            <artifactId>org.openl.rules.xls.merge</artifactId>
+            <version>${project.version}</version>
```

Comment:
Move version management to the root pom.xml

Reply:
Done